packages/ckeditor5-table/tests/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -1946,6 +1946,104 @@ describe( 'TableColumnResizeEditing', () => {
 		} );
 	} );
 
+	describe( 'Hide/show resizers', () => {
+		let model, editor, view, editorElement;
+
+		beforeEach( async () => {
+			editorElement = document.createElement( 'div' );
+			document.body.appendChild( editorElement );
+			editor = await createEditor();
+
+			model = editor.model;
+			view = editor.editing.view;
+		} );
+
+		afterEach( async () => {
+			if ( editorElement ) {
+				editorElement.remove();
+			}
+
+			if ( editor ) {
+				await editor.destroy();
+			}
+		} );
+
+		describe( 'on mousedown/mouseup', () => {
+			it( 'should not hide resizers when resizer is clicked', () => {
+				setModelData( model,
+					'<table columnWidths="100%">' +
+					'<tableRow>' +
+					'<tableCell>' +
+					'<paragraph>' +
+					'[foo]' +
+					'</paragraph>' +
+					'</tableCell>' +
+					'</tableRow>' +
+					'</table>'
+				);
+
+				const resizer = view.getDomRoot().querySelector( '.table-column-resizer' );
+
+				resizer.dispatchEvent( new MouseEvent( 'mousedown', { bubbles: true } ) );
+
+				const viewRoot = view.domConverter.mapDomToView( view.getDomRoot() );
+
+				expect( viewRoot.hasClass( 'ck-resizers-hidden' ) ).to.be.false;
+			} );
+
+			it( 'should hide resizers when table cell is clicked and show resizers on mouseup', () => {
+				setModelData( model,
+					'<table columnWidths="100%">' +
+					'<tableRow>' +
+					'<tableCell>' +
+					'<paragraph>' +
+					'[foo]' +
+					'</paragraph>' +
+					'</tableCell>' +
+					'</tableRow>' +
+					'</table>'
+				);
+
+				const td = view.getDomRoot().querySelector( 'td' );
+
+				td.dispatchEvent( new MouseEvent( 'mousedown', { bubbles: true } ) );
+
+				const viewRoot = view.domConverter.mapDomToView( view.getDomRoot() );
+
+				expect( viewRoot.hasClass( 'ck-resizers-hidden' ) ).to.be.true;
+
+				td.dispatchEvent( new MouseEvent( 'mouseup', { bubbles: true } ) );
+
+				expect( viewRoot.hasClass( 'ck-resizers-hidden' ) ).to.be.false;
```

Comment:
After we split the tests this will be irrelevant, but having a few boolean `expect`s without messages will cause that if this test fails, you won't know which expect is responsible.

Ok, here you'd still know because one of them should be `true` and the other `false`, but you get the point.